classes/book.rb

```diff
@@ -0,0 +1,11 @@
+# Define class Book
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ]  Great job for creating Book class, but seemed that you might have forgetted to add `add_rental method`. This method should present in your book class as well to help you to add both person and date when the book is rented, for more [check ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`)